### PR TITLE
MES-1250 - Competency Button

### PR DESCRIPTION
**Pull request by Cavalier07**
## Description and relevant Jira numbers

Pretty button to use for competencies on the test report

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [x] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [x] Any new reusable component/widget added to component library on Confluence

- [x] Screenshot(s) captured on the physical device (if applicable)
- [x] Tested by QA
- [x] PO's approval

UNADJUSTEDNONRAW_thumb_46

### CODE REVIEW

**Comment by ammarhaiderbjss** [comment]
@Cavalier07 Screenshot pls?

**Comment by Cavalier07** [comment]
> @Cavalier07 Screenshot pls?

You beat me to uploading it - It's there now

File: src/pages/test-report/test-report.ts [hunk 1]
```diff
@@ -31,14 +31,18 @@ export class TestReportPage extends BasePageComponent {
...
+      this.insomnia.keepAwake();
+    }
   }
 
   ionViewDidLeave(): void {
-    this.deviceProvider.disableSingleAppMode();
-    this.screenOrientation.unlock();
-    this.insomnia.allowSleepAgain();
+    if (super.isIos) {
+      this.deviceProvider.disableSingleAppMode();
```
Comment (ammarhaiderbjss) [inline src/pages/test-report/test-report.ts]: Can we add some `expect` statements to the existing test to cover these bits

File: src/pages/test-report/test-report.ts [hunk 2]
```diff
@@ -31,14 +31,18 @@ export class TestReportPage extends BasePageComponent {
 
   ionViewDidEnter(): void {
     this.store$.dispatch(new TestReportViewDidEnter());
-    this.deviceProvider.enableSingleAppMode();
-    this.screenOrientation.lock(this.screenOrientation.ORIENTATIONS.PORTRAIT_PRIMARY);
-    this.insomnia.keepAwake();
+    if (super.isIos()) {
+      this.deviceProvider.enableSingleAppMode();
```
Comment (ammarhaiderbjss) [inline src/pages/test-report/test-report.ts]: Same here

**Pull request merged**